tests/phpunit/tests/block-supports/shadow.php

```diff
@@ -35,55 +40,71 @@ public function test_shadow_style_is_applied() {
 						'type' => 'object',
 					),
 				),
-				'supports'    => array(
-					'shadow' => true,
-				),
+				'supports'    => $supports,
 			)
 		);
-		$registry   = WP_Block_Type_Registry::get_instance();
-		$block_type = $registry->get_registered( $this->test_block_name );
-		$block_atts = array(
-			'style' => array(
-				'shadow' => '60px -16px teal',
-			),
-		);
+		$registry = WP_Block_Type_Registry::get_instance();
 
-		$actual   = wp_apply_shadow_support( $block_type, $block_atts );
-		$expected = array(
-			'style' => 'box-shadow:60px -16px teal;',
+		return $registry->get_registered( $this->test_block_name );
+	}
+
+	/**
+	 * Tests the generation of shadow block support styles.
+	 *
+	 * @dataProvider data_generate_shadow_fixtures
+	 *
+	 * @param boolean|array $support Shadow block support configuration.
+	 * @param string        $value   Shadow style value for style attribute object.
+	 * @param array         $expected       Expected shadow block support styles.
+	 */
+	public function test_wp_apply_shadow_support( $support, $value, $expected ) {
```

Comment:
@tellthemachines why the ticket number is not mention for test?

Reply:
Thanks for spotting that! Fixed in https://core.trac.wordpress.org/changeset/58315.